.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,62 @@
+name: 🐛 Bug Report
+description: Create a report to help us reproduce and fix the bug
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      #### Before submitting a bug, please make sure the issue hasn't been already addressed by searching through [the existing and past issues](https://github.com/pytorch/audio/issues?q=is%3Aissue+sort%3Acreated-desc+).
+- type: textarea
+  attributes:
+    label: 🐛 Describe the bug
+    description: |
+      Please provide a clear and concise description of what the bug is.
+
+      If relevant, add a minimal example so that we can reproduce the error by running the code. It is very important for he snippet to be as succinct (minimal) as possible, so please take time to trim down any irrelevant code to help us debug efficiently. We are going to copy-paste your code and we expect to get the same result as you did: avoid any external data, and include the relevant imports, etc. For example:
```

```suggestion
      If relevant, add a minimal example so that we can reproduce the error by running the code. It is very important for the snippet to be as succinct (minimal) as possible, so please take time to trim down any irrelevant code to help us debug efficiently. We are going to copy-paste your code and we hope to get the same result as you did: avoid any external data, and include the relevant imports, etc. For example:
```

IMHO, `expect` is a bit strong and I am afraid this might scar some users away because it might suggest that we will not respond if it's the snippet is runnable.
Of course, the complete repro code is nice and helpful and often times, a good gist is also as efficient as well.
We cannot always expect the complete repro code for various reasons, yet we still would like to see an issue filed than not filed at all.